Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -0,0 +1,463 @@
+  , gravity CENTER
+  ][  linearLayout
+      [ width MATCH_PARENT
+      , height WRAP_CONTENT
+      , gravity RIGHT
+      , layoutGravity "right"
+      , padding $ PaddingTop safeMarginTop
+      ][  imageView
+          [ height $ V 40
+          , width $ V 40
+          , accessibility ENABLE
+          , accessibilityHint "Contact Support : Button"
+          , imageWithFallback $ "ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ic_headphone.png"
+          , onClick push $ const Support
+          ]
+      ]
+    , if config.topCard.topPill.visible then topPillView config push else dummyTextView
+    , linearLayout
+      [ width MATCH_PARENT
+      , weight 1.0
+      , orientation VERTICAL
+      , gravity CENTER
+      ][  textView $
+          [ width WRAP_CONTENT
+          , height WRAP_CONTENT
+          , text config.topCard.title
+          , color Color.grey900
+          ] <> FontStyle.h1 TypoGraphy
+        , linearLayout
+          [ width WRAP_CONTENT
+          , height WRAP_CONTENT
+          , gravity CENTER
+          ][ textView
+              [ text $ "₹" <> (show config.topCard.finalAmount)
+              , accessibilityHint $ "Ride Complete: Final Fare ₹"  <> (show config.topCard.finalAmount)
+              , accessibility ENABLE
+              , color Color.white900
+              , textSize FontSize.a_72
+              , width WRAP_CONTENT
+              , height WRAP_CONTENT
+              , fontStyle $ FontStyle.bold LanguageStyle
+              ]
+          , textView
+              [ textFromHtml $ "<strike> ₹" <> (show config.topCard.initalAmount) <> "</strike>"
+              , accessibilityHint $ "Your Fare Has Been Updated From ₹"  <> (show config.topCard.initalAmount) <> " To ₹" <> (show config.topCard.finalAmount)
+              , accessibility ENABLE
+              , textSize FontSize.a_28
+              , margin $ Margin 8 5 0 0
+              , width WRAP_CONTENT
+              , height WRAP_CONTENT
+              , fontStyle $ FontStyle.medium LanguageStyle
+              , lineHeight "40"
+              , color Color.black600
+              , visibility if config.topCard.fareUpdatedVisiblity then VISIBLE else GONE
+              ]
+          ]
+        , pillView config push
+
+      ]
+    , linearLayout
+      [ width MATCH_PARENT
+      , height $ V 1
+      , background Color.black800
+      ][]
+    , linearLayout
+      [ width MATCH_PARENT
+      , height WRAP_CONTENT
+      , margin $ MarginTop 10
+      , gravity CENTER_VERTICAL
+      , onClick push $ const RideDetails
+      , accessibility ENABLE
+      , accessibilityHint "Ride Details : Button"
+      ][  textView
+          [ height WRAP_CONTENT
+          , text config.topCard.bottomText
+          , color Color.white900
+          , weight 1.0
+          , textSize FontSize.a_14
+          , fontStyle $ FontStyle.medium TypoGraphy
```

Comment:
change it in all the places append the font props. Take reference from other components